### PR TITLE
Week 7 Documentation

### LINKED ISSUE #22: Week 7: Update DG - Appendix: Requirements - Product scope + User Stories
Update:
1. Target user profile
2. value proposition
3. user stories

[Read for Details](https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#5-update-the-dg-user-stories-glossary-nfrs-use-cases)

### PR DESCRIPTION
**Pull request by Eugene-Ong-W-X**
Done
1. PPP skeleton
2. Profile Pic (placeholder image - for privacy reasons)

To be done
1. UG
2. DG

### CODE REVIEW

**Review by PangKuangWei — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -255,30 +255,55 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* *`    | user                    | tag a contact                                              | add it to a category                                                       |
+| `* *`    | user                    | untag a contact                                            | remove it from a category                                                  |
+| `* *`    | user                    | search using a name, phone number, email or address        | find a contact easily without reading through the list                     |
+| `* *`    | user                    | search with a tag                                          | find groups of contacts that share a common tag                            |
+| `* *`    | user                    | search with multiple tags                                  | narrow my search results to only contacts that have all the specified tags |
+| `* *`    | user                    | search with multiple tags                                  | broaden my result results to contacts that have any of the specified tags  |
```
Comment (Ugholaf) [inline docs/DeveloperGuide.md:294]: These two user stories sound a bit contradicting to me, maybe just the first one will be good enough, what do you think?

Comment (Eugene-Ong-W-X) [reply docs/DeveloperGuide.md:294]: These are two different functions for the improved search feature
1. matches contacts with **all** the specified tags
2. matches contacts with **any** of the specified tags (any = can be 1, can be multiple, can be all the tags)

Comment (Ugholaf) [reply docs/DeveloperGuide.md:294]: okay now it makes sense, I didn't see the all and any. Thanks for clarifying.

**Pull request merged**